libraries/aws_cloudformation_stacks.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsCloudformationStacks < AwsResourceBase
+  name 'aws_cloudformation_stacks'
+  desc 'Verifies settings for aws CloudFormation Stacks'
+
+  example "
+    describe aws_cloudformation_stacks do
+      it { should exist }
+    end
+  "
+
+  attr_reader :table
+
+  FilterTable.create
+             .register_column(:names, field: :name)
+             .register_column(:creation_times, field: :creation_time)
+             .register_column(:stack_ids, field: :stack_id)
+             .register_column(:notification_arns, field: :notification_arn)
+             .register_column(:role_arns, field: :role_arn)
+             .register_column(:parent_ids, field: :parent_id)
+             .register_column(:root_ids, field: :root_id)
+             .install_filter_methods_on_resource(self, :table)
+
+  def initialize(opts = {})
+    super(opts)
+    validate_parameters
+    @table = fetch_data
+  end
+
+  def fetch_data
+    cloudformation_stacks_rows = []
+    pagination_options = {}
+    loop do
+      catch_aws_errors do
+        @api_response = @aws.cloudformation_client.describe_stacks(pagination_options)
+      end
+      return [] if !@api_response || @api_response.empty?
```

Comment:
also shouldn't this be returning `cloudformation_stacks_rows` ? consider if the second loop returns an empty response for whatever reason you would still have the first paged response..

Reply:
Done